# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /home/ondra/netbeans/cdev/jira/test/unit/src/org/netbeans/modules/jira/issue
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: NbJiraIssueTest.java
--- NbJiraIssueTest.java Base (BASE)
+++ NbJiraIssueTest.java Locally Modified (Based On LOCAL)
@@ -38,6 +38,8 @@
  */
 package org.netbeans.modules.jira.issue;
 
+import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
 import org.eclipse.core.runtime.CoreException;
 import org.netbeans.modules.jira.*;
 import java.util.logging.Level;
@@ -117,7 +119,7 @@
         //getClient().getCache().refreshDetails(JiraTestUtil.nullProgressMonitor);
         config = getRepository().getConfiguration();
         config.refreshDetails(new NullProgressMonitor());
-        JiraTestUtil.cleanProject(getRepositoryConnector(), getRepository().getTaskRepository(), getClient(), config.getProjectByKey(TEST_PROJECT));
+//        JiraTestUtil.cleanProject(getRepositoryConnector(), getRepository().getTaskRepository(), getClient(), config.getProjectByKey(TEST_PROJECT));
     }
 
     @Override
@@ -174,6 +176,18 @@
         assertEquals(commentCount++, comments.length);
     }
 
+    public void testAddAttachments () {
+        NbJiraIssue issue = (NbJiraIssue) repository.getIssue("NBJIRAISSUEDEV-137");
+        assertNotNull(issue);
+        NbJiraIssue.Attachment attachments[] = issue.getAttachments();
+        assertNotNull(attachments);
+        assertTrue(attachments.length > 0);
+        ByteArrayOutputStream os = new ByteArrayOutputStream(Integer.parseInt(attachments[0].getSize()));
+        attachments[0].getAttachementData(os);
+        byte[] attContent = os.toByteArray();
+        
+    }
+
     private NbJiraIssue createIssue() throws CoreException {
         NbJiraIssue issue = (NbJiraIssue) getRepository().createIssue();
 
